odoo/addons/base/models/assetsbundle.py

```diff
@@ -722,6 +817,21 @@ def to_node(self):
                 ['data-asset-version', self.bundle.version],
             ]), self.with_header())
 
+    def with_header(self, content=None, minimal=True):
+        if minimal:
+            return super(JavascriptAsset, self).with_header(content)
+        else:
+            header = [
+                "*  Filepath: %s" % self.url,
+                "*  Bundle: %s" % self.bundle.name,
+                "*  Lines: %s" % len(content.splitlines()),
+            ]
+            length = max([len(line) for line in header])
```

Comment:
Usually I'm all for comprehensions but here it's really a perfect use-case for `map`:
```suggestion
            length = max(map(len, header))
```